Readme.md

```diff
@@ -30,21 +24,21 @@ To add a new protocol, you will need to complete the following high-level steps:
 #### Native protocol:
 
 1. Create a protocol state struct that implements the `ProtocolSim` trait.
-2. Create a tycho decoder for the protocol state: i.e. implement `TryFrom` for `ComponentWithState` to your new 
-protocol state.
+2. Create a tycho decoder for the protocol state: i.e. implement `TryFrom` for `ComponentWithState` to your new
+   protocol state.
 
 Each native protocol should have its own module under `tycho-simulation/src/protocol`.
 
 #### VM protocol:
 
 1. Add the associated adapter runtime file to `tycho-simulations/src/protocol/assets`. Make sure to name the file
-according to the protocol name used by tycho in the following format: `<Protocol><Version>Adapter.evm.runtime`. 
-For example: `vm:balancer_v2` will be `BalancerV2Adapter.evm.runtime`.
+   according to the protocol name used by tycho in the following format: `<Protocol><Version>Adapter.evm.runtime`.
+   For example: `vm:balancer_v2` will be `BalancerV2Adapter.evm.runtime`.
```

Comment:
Should we at some point link to the gitbook docs?
This piece of information is pretty much useless without knowing how to generate this runtime file

Reply:
we need to publish the docs before linking the docs though. but yes I agree